src/ParallelAlgorithms/Interpolation/Events/Interpolate.hpp

```diff
@@ -50,6 +50,7 @@ template <size_t VolumeDim, typename InterpolationTargetTag,
           typename... InterpolatorSourceVarTags>
 class Interpolate<VolumeDim, InterpolationTargetTag,
                   tmpl::list<InterpolatorSourceVarTags...>> : public Event {
+ public:
```

Comment:
Worked before because `WRAPPED_PUPable_decl_template` contains a `public:`, but we should definitely not be relying on that.